examples/matlab_mex/pendulum_on_cart_model/example_closed_loop.m

```diff
@@ -45,56 +43,48 @@
 end
 
 
-
-%% arguments
-compile_mex = 'true';
-codgen_model = 'true';
+%% options
+compile_mex = 'true'; % true, false
+codgen_model = 'true'; % true, false
 % simulation
-sim_method = 'irk';
-sim_sens_forw = 'false';
+sim_method = 'irk'; % erk, irk, irk_gnsf
+sim_sens_forw = 'false'; % true, false
 sim_num_stages = 4;
 sim_num_steps = 4;
 % ocp
 param_scheme = 'multiple_shooting_unif_grid';
 ocp_N = 100;
-nlp_solver = 'sqp';
```

Comment:
All these options commented out were good to quickly switch between them while developing, without having to type them all time time... I would keep them there...

Reply:
These examples are what we point the end users to. I think they should be as clear and compact as possible.
I was documenting each option with all possible values, e.g.
` sim_method = 'irk'; % erk, irk, irk_gnsf`
It is one line -> compact.
It lists all possible options -> clear.
Switching between options is still fairly easy, just copy a string in the comment behind and replace the one in code.

Reply:
10 lines more in the file are not going to make any difference about compactness, but anything that takes longer than commenting in or out a line is annoying for development.

That is the reason why that stuff exists at the beginning of the examples. It is not part of the interface, and actually the correct place to list all possible options is not there, but in the actual setter, e.g. https://github.com/acados/acados/blob/master/examples/matlab_mex/pendulum_on_cart_model/example_closed_loop.m#L222